<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-KK1SMTI" failures="1" tests="2" name="mytest1(failed)" time="73.034" errors="0" timestamp="2024-11-20T22:41:37 IST">
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verifytest" time="13.629">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_003_LoginDataDrivenTest.verifytest(TC_003_LoginDataDrivenTest.java:41)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifytest -->
  <testcase classname="testCases.TC_004_ForgetPasswordTest" name="forgetpassword" time="4.076"/>
</testsuite> <!-- mytest1(failed) -->
